package.json

```diff
@@ -80,7 +80,7 @@
     "ts-node": "3.2.0",
     "tslint": "5.3.2",
     "tslint-config-prettier": "1.6.0",
-    "typescript": "2.4.2",
+    "typescript": "2.6.2",
```

Comment:
Any specific reason to update to 2.6.2?

Reply:
@jsonberry I can revert, but the new version is better, faster and stricter